src/app/home/countdown/countdown.component.scss

```diff
@@ -0,0 +1,64 @@
+#countdown-title {
+  color: #64b5f6;
+}
+
+.card-content {
+  background: #64b5f6;
+}
+
+.title-font {
+  font-size: 20px;
+}
+
+.medium-text {
+  font-size: 25px;
+}
+
+.number-style-mobile {
+  font-size: 20px;
+
+  @media screen and (max-width: 415px) {
```

Comment:
For media queries we are using the default ones defined in materialize variables, these are under: `materialize-css/sass/components/variables` and you can see an example of use in the navbar.component.scss